.github/workflows/openstack/reboot_watch

```diff
@@ -10,21 +10,44 @@ use POSIX;
 my $RETVAL  = 1;
 my $RETRIES = 0;
 
+# Verify REGEX isn't already in the log before we go into tail mode.  The logodump here is sgnificantly faster than File::Tail.
+# Jump if it's found.
+
+exit 1 unless defined $ENV{REGEX};
+
+open my $elevate_log_fh, ELEVATE_LOG_PATH or die "## [ERROR]: Unable to open ELEVATE_LOG_PATH: $!\n";
+
+while ( my $line = readline $elevate_log_fh ) {
+    if ( $line =~ m/\Q${REGEX}\E/ ) {
```

Comment:
This would be safer to be an index check, NOT a regex.

Reply:
```perl
index($line, $ENV{REGEX}) >= 0
```

Reply:
Implemented